UDV-Core/src/Modules/ConsultDoc/DocumentResearch.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Class: DocumentResearch
+ * Description :
+ * The DocumentResearch is an object handling the research view
+ *
+ */
+
+import $ from 'jquery'; //to use Alpaca
+import 'alpaca';  //provides a simple way to generate HTML forms using jQuery
+
+/**
+ *
+ * @constructor
+ * @param { HTML DOM Element object } researchContainer
+ * @param { documentController } documentController
+ */
+
+export function DocumentResearch(researchContainer, documentController)
+{
+    //Class attributes
+    this.documentController = documentController;
+    this.researchController = researchContainer;
+    this.windowIsActive = false;
+
+    /**
+     * Creates the research view
+     */
+    //=============================================================================
+    this.initialize = function initialize()
+    {
+        this.researchController.innerHTML =
+            '<div id = "filtersTitle">Document research</div>\
+            <button id = "closeResearch">X</button>\
+            <div id = "filtersWindow"></div>\
+            <div id ="researchWindowTabs">\
+            <button id = "docResearch">Search</button>\
+            </div>\
+            ';
+
+        var optionsFilter = "http://rict.liris.cnrs.fr/optionsFilter.json";
+        var schema = "http://rict.liris.cnrs.fr/schemaType.json";
```

Comment:
The number of configuration files is growing up, I think that it could be a good idea to list them (along with their roles) and to provide examples in a dedicated page pointed in the README.md.

Reply:
The Readme.md is in UDV-Core/src/Modules/ConsultDoc